src/Factories/EnvironmentSetup/ProductionEnvironmentSetup.php

```diff
@@ -9,23 +9,30 @@
 use Monolog\Handler\StreamHandler;
 use Monolog\Logger;
 use Psr\Log\LoggerInterface;
+use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+use Symfony\Component\Cache\Psr16Cache;
 use WMDE\Fundraising\Frontend\Factories\FunFunFactory;
 
 class ProductionEnvironmentSetup implements EnvironmentSetup {
 
 	public function setEnvironmentDependentInstances( FunFunFactory $factory ) {
 		$this->initializeLoggers( $factory );
-
-		$factory->enableCaching();
 	}
 
 	private function initializeLoggers( FunFunFactory $factory ) {
+		$this->setCampaignCache( $factory );
 		$this->setPaypalLogger( $factory );
 		$this->setSofortLogger( $factory );
 		$this->setCreditCardLogger( $factory );
 		$this->setDoctrineConfiguration( $factory );
 	}
 
+	private function setCampaignCache( FunFunFactory $factory ) {
+		$factory->setCampaignCache(
+			new Psr16Cache( new FilesystemAdapter( 'campaigns', 3600, $factory->getCachePath() . '/campaigns' ) )
```

Comment:
Since we need to deploy for new campaign files anyway, you can set the campaign cache to 24 hours or even 1 week